### PR TITLE
update readme with basic usage

### PR DESCRIPTION
**Pull request by digitronik**
cc @omkarkhatavkar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
-# webdriver_kaifuku
-restartable webdriver instances
+<h1 align="center"> webdriver_kaifuku </h1>
+<h3 align="center"> Restartable webdriver instances </h3>
```
Comment (RonnyPfannschmidt) [inline README.md:2]: Why html?

Comment (digitronik) [reply README.md:2]: I can say copy from my other repo

but If I remember correctly; I used HTML for central alignment. 
I think markdown or rst no support such styling.

https://github.com/RonnyPfannschmidt/webdriver_kaifuku/blob/869963993b299afc2df003a4f8870010ab0ebf7d/README.md

@RonnyPfannschmidt I was inspired by psf `black` project.

Comment (digitronik) [reply README.md:2]: @RonnyPfannschmidt can I change it to markdown?

Comment (RonnyPfannschmidt) [reply README.md:2]: thanks for the ping, no its fine

**Pull request merged**